src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PotentialEditorConfigFiles.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="PotentialEditorConfigFiles"
+      Description="Potential .editorconfig files"
```

Comment:
Remove, this file will never appear in UI.

Reply:
Done.